modules/customize-certificates-add-service-serving-mutating-webhook.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-mutating-webhook_{context}"]
+= Add the service CA bundle to a MutatingWebhookConfiguration
+
+You can annotate a MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+.Procedure
+
+. Annotate the MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true`.
```

Comment:
s/true`./true`:/
Following convention of using a colon in the rest of the procedure